### ISSUE TITLE
Add a method for users to be able to see who voted on a post, and in what way

### ISSUE THREAD

**Issue by K4LCIFER** [LemmyNet/lemmy-ui#2182]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

From what I understand, KBin users are able to see who voted on a post. It would be nice if users of the lemmy browser UI could also see this.

It could function by something along the lines of a toggleable list that displays all the users that upvoted, and all the users that downvoted.

**Comment by Nutomic** [comment]
This is available for admins and mods.

**Comment by Dakkaron** [comment]
Sorry for the necropost. I think it would be good to make this available for users as well.

My main reasoning for that is that it's currently not clear to users that votes aren't anonymous.

Regular users can use kbin or [lemvotes](https://github.com/gragorther/votes) to easily see who voted for a certain piece of content.

That might come as quite a surprise to users who are used to votes being anonymous.

In fact, I have seen moderators and admins issuing bans for downvotes, so it should really be obvious to regular users that votes aren't anonymous at all.

**Comment by dessalines** [comment]
No, [see the discussion here](https://github.com/LemmyNet/lemmy/issues/4967)

**Comment by Dakkaron** [comment]
Might be worth reopening the discussion, since lemvotes and kbin have changed one of the main points in the discussion.

Over and over people were talking about how the barrier of creating your own instance to view votes is so high that the votes are practically anonymous.

But now that's not the case. Lemvotes allows you to just copy and paste the URL of a post or comment into a ready-made site and you can see the votes. Alternatively, you can just open the post in kbin.

The kbin situation makes this even more problematic, because a lemmy user might think their votes are anonymous while a kbin user can just see them without having to use any tools at all.

At least the Lemmy UI should have a clear information somewhere visible (e.g. next to the post vote buttons) that posts are not anonymous and that it's trivial to view them.

I'd prefer votes to be fully anonymous, or at least fully anonymous to non-admins/non-mods. Since this is technically impossible, I'd like that it's clear to everyone that their votes aren't anonymous.

**Comment by dessalines** [comment]
I encourage all servers to block lemvotes (many already do), as its a snooping application and allows any random person to spy on your votes. Essentially its an admin-only server made to spy on users.

I think the legal section or something is the only way to tell ppl that admins of instances can see their votes. I'd rather not have it clutter up the UI.

**Comment by Dakkaron** [comment]
What about a tooltip? Doesn't work for mobile, of course, but at least for desktop it would work.

I fear the legal section is something that nobody reads...

Blocking lemvotes makes sense, but as long as there's one instance that doesn't block it, that's enough to be able to use it.

Would you also encourage to block kbin, which has this feature built-in?

**Comment by dessalines** [comment]
kbin doesn't exist solely for the purpose of snooping on users votes, and its ultimately up to them whether they want to display it, but we've opened up an issue over there about how lemmy's userbase overwhelmingly doesn't want their votes displayed to randos.

> What about a tooltip? Doesn't work for mobile, of course, but at least for desktop it would work.

PRs welcome there, but IMO its not necessary.
